tools/assetpipelineeditor/animationwidget.cpp

```diff
@@ -304,7 +304,7 @@ void AnimationClipModel::updateDecorationIcon()
     static QPixmap playPixmap(":/icons/playing-circleH.png");
```

Comment:
wondering if this wouldn't benefit from being static const

Reply:
done